src/EnergyPlus/SingleDuct.cc

```diff
@@ -2052,43 +2039,43 @@ namespace SingleDuct {
         int OutletNode;
         int SysIndex;
         static bool ZoneEquipmentListChecked(false); // True after the Zone Equipment List has been checked for items
-        static Array1D_bool MyEnvrnFlag;
-        static Array1D_bool MySizeFlag;
-        static Array1D_bool GetGasElecHeatCoilCap; // Gets autosized value of coil capacity
+        //static Array1D_bool MyEnvrnFlag;
+        //static Array1D_bool MySizeFlag;
+        //static Array1D_bool GetGasElecHeatCoilCap; // Gets autosized value of coil capacity
```

Comment:
Ideally these would be deleted, but I'm willing to let it go if nothing else needs to change.

Reply:
@Myoldmopar I was anticipating for comments related to getInput() function that may require changes; otherwise, I don't have much to add at this point.